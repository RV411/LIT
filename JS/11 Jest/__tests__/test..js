//! Snapshot

const persona={
    nombre:'Aaron',
    balance:500,
    tipo:'Premium'
}

describe('Test',()=>{
    test('Es Aaron',()=>{
        expect(persona).toMatchSnapshot();
    })
})