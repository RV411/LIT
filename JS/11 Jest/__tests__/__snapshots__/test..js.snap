// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Es Aaron 1`] = `
{
  "balance": 500,
  "nombre": "Aaron",
  "tipo": "Premium",
}
`;
